src/main.nim

```diff
@@ -5480,6 +5489,15 @@ proc handleLevelMouseEvents(a) =
 
     else: discard
 
+# }}}
+
+# {{{ setSelectJumptoLinkActionMessage()
+proc setSelectJumptoLinkActionMessage(a) =
+  let currIx = a.ui.jumpToDestinationIx + 1
+  let count = a.ui.jumpToDestinations.len
+  setStatusMessage(NoIcon, fmt"Select destination ({currIx} of {count})",
```

Comment:
So we're selecting between multiple source cells here, not destinations. Also, it would be nice to include the type of the link in the message like in other places.

```nim
fmt"Select {linkFloorToString(floor)} source ({currIx} of {count})"
```